src/Uno.Toolkit.UI/Controls/AutoLayout/AutoLayout.cs

```diff
@@ -393,9 +398,9 @@ private void UpdateAlignments()
 					//We need to make sure that the independent layout can span all across his parent
 					_grid.RowDefinitions.Add(new RowDefinition() { Height = new GridLength(1, GridUnitType.Star) });
 				}
-				if (hasPadding)
+				if (hasPadding && padding.Bottom - spacing > 0)
```

Comment:
I don't see a runtime test :) You should be able to build it now.

Reply:
I will add one in the next commit to unblock Isaac as soon as possible.